modules/renter/contractor/downloader.go

```diff
@@ -145,10 +170,11 @@ func (c *Contractor) Downloader(id types.FileContractID) (_ Downloader, err erro
 
 	// cache downloader
 	hd := &hostDownloader{
-		clients:    1,
-		contractID: contract.ID,
-		contractor: c,
-		downloader: d,
+		clients:      1,
+		contractID:   contract.ID,
+		contractor:   c,
+		downloader:   d,
+		hostSettings: host.HostExternalSettings,
```

Comment:
including the whole `Host` might be best, especially for things like debugging. easy to change later though